libs/designsystem/checkbox/src/checkbox.component.ts

```diff
@@ -51,9 +60,59 @@ export class CheckboxComponent {
 
   onChecked(checked: boolean): void {
     this.checked = checked;
+    this._onChange(this.checked);
     this.checkedChange.emit(this.checked);
+    this._onTouched();
   }
 
   // IDs used for a11y labelling
   _labelId = UniqueIdGenerator.scopedTo('kirby-checkbox-label').next();
+
+  // Initialize empty ControlValueAccessor callback functions
+  // eslint-disable-next-line no-empty-function
+  private _onChange: (value: boolean) => void = () => {};
+  // eslint-disable-next-line no-empty-function
+  private _onTouched = () => {};
+
+  /**
+   * Sets the select's value. Part of the ControlValueAccessor interface
+   * required to integrate with Angular's core forms API.
+   *
+   * @param value New value to be written to the model.
+   */
+  writeValue(value: boolean): void {
+    this.checked = value;
+  }
+
+  /**
+   * Saves a callback function to be invoked when the select's value
+   * changes from user input. Part of the ControlValueAccessor interface
+   * required to integrate with Angular's core forms API.
+   *
+   * @param fn Callback to be triggered when the value changes.
+   */
+  registerOnChange(fn: any): void {
+    this._onChange = fn;
+  }
+
+  /**
+   * Saves a callback function to be invoked when the select is blurred
+   * by the user. Part of the ControlValueAccessor interface required
+   * to integrate with Angular's core forms API.
+   *
+   * @param fn Callback to be triggered when the component has been touched.
+   */
+  registerOnTouched(fn: any): void {
+    this._onTouched = fn;
+  }
+
+  /**
+   * Disables the select. Part of the ControlValueAccessor interface required
```

Comment:
Copy pasted from somehere? ;)
```suggestion
   * Disables the checkbox. Part of the ControlValueAccessor interface required
```